Orange/widgets/data/owfile.py

```diff
@@ -94,20 +117,40 @@ def __init__(self):
         self.file_combo.setSizePolicy(Policy.MinimumExpanding, Policy.Fixed)
         self.file_combo.activated[int].connect(self.select_file)
         box.layout().addWidget(self.file_combo)
-        button = gui.button(
-            box, self, '...', callback=self.browse_file, autoDefault=False)
-        button.setIcon(self.style().standardIcon(QtGui.QStyle.SP_DirOpenIcon))
-        button.setSizePolicy(Policy.Maximum, Policy.Fixed)
-        button = gui.button(
-            box, self, "Reload", callback=self.reload, autoDefault=False)
-        button.setIcon(self.style().standardIcon(QtGui.QStyle.SP_BrowserReload))
-        button.setSizePolicy(Policy.Fixed, Policy.Fixed)
-        layout.addWidget(box, 0, 1,  QtCore.Qt.AlignVCenter)
+        layout.addWidget(box, 0, 1)
+
+        fileButton = gui.button(
+            None, self, '...', callback=self.browse_file, autoDefault=False)
+        fileButton.setIcon(self.style().standardIcon(
+            QtGui.QStyle.SP_DirOpenIcon))
+        fileButton.setSizePolicy(Policy.Maximum, Policy.Fixed)
+        layout.addWidget(fileButton, 0, 2)
+
+        reloadButton = gui.button(
+            None, self, "Reload", callback=self.reload, autoDefault=False)
+        reloadButton.setIcon(self.style().standardIcon(
+            QtGui.QStyle.SP_BrowserReload))
+        reloadButton.setSizePolicy(Policy.Fixed, Policy.Fixed)
+        layout.addWidget(reloadButton, 0, 3)
+
+        self.hBLayout = gui.hBox(None, addToLayout=False, margin=0)
+        self.sheet_combo = gui.comboBox(None, self, "xls_sheet",
+                                        sendSelectedValue=True)
+        self.sheet_combo.setSizePolicy(
+            Policy.MinimumExpanding, Policy.Fixed)
+        self.sheet_combo.activated[str].connect(self.select_sheet)
```

Comment:
Remove this line. Attributes must be handled by `gui`'s callback, otherwise context settings do not work out of the box.